### PR TITLE
EJDOTNETCORE-4253 Added readme file in PDF-A4 conformance

### PR DESCRIPTION
**Pull request by sameerkhan001**
Hi sir,

Added read me file in PDF-A4 conformance using C# 

Regards,
Sameerkhan N

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
 # Create PDF/A-4 Conformance Documents using C#
-This repository contains Syncfusion PDF library examples that explains how to create PDF/A-4 conformance and convert normal PDF to PDF/A-4 conformance document.
 
-## Blog reference
-[Create PDF/A-4 Documents for Long-Term Archiving in C#](https://www.syncfusion.com/blogs/post/create-pdf-a-4-documents-for-long-term-archiving-in-c.aspx)
+This repository contains [Syncfusion PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) examples that explains how to create PDF/A-4 conformance and convert normal PDF to PDF/A-4 conformance document.
```
Comment (ghost) [inline README.md]: Add intro about Syncfusion .NET PDF library

Comment (ghost) [inline README.md]: This repository contains examples of creating PDF/A-4 conformance documents and converting PDF to PDF/A-4 document.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+
+ Sample name | Description
+------|------
+[PDF/A-4e](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_4e_conformance) |  PDF/A-4e (engineering) is intended for technical documents and replaces PDF/E format. It supports 3D models, rich media, 3D annotations, and embedded files. 
+ [PDF/A-4f](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_4f_conformance) | PDF/A-4f allows us to embed any files in PDF documents as attachments similar to the PDF/A-3 conformance level. 
+[ PDF to a PDF/A-4](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_to_Pdf-A4) |You can easily convert an existing PDF to a PDF/A-4 document for archiving and long-term preservation.
```
Comment (ghost) [inline README.md]: Convert an existing PDF document to PDF/A-4 document for archiving and long-term preservation.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+This repository contains [Syncfusion PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) examples that explains how to create PDF/A-4 conformance and convert normal PDF to PDF/A-4 conformance document.
+
+ Sample name | Description
+------|------
+[PDF/A-4e](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_4e_conformance) |  PDF/A-4e (engineering) is intended for technical documents and replaces PDF/E format. It supports 3D models, rich media, 3D annotations, and embedded files. 
+ [PDF/A-4f](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_4f_conformance) | PDF/A-4f allows us to embed any files in PDF documents as attachments similar to the PDF/A-3 conformance level. 
+[ PDF to a PDF/A-4](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_to_Pdf-A4) |You can easily convert an existing PDF to a PDF/A-4 document for archiving and long-term preservation. 
+[PDF/A-4](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/conformance_creation) | You can easily create a new document with the PDF/A-4 standard.
```
Comment (ghost) [inline README.md]: Create a new PDF document with PDF/A-4 standard

File: README.md (hunk 1 above)
Comment (sameerkhan001) [reply README.md]: changes Updated

File: README.md (hunk 2 above)
Comment (sameerkhan001) [reply README.md]: Changes Updated

File: README.md (hunk 3 above)
Comment (sameerkhan001) [reply README.md]: Changes Updated

File: README.md (hunk 1 above)
Comment (sameerkhan001) [reply README.md]: Added

File: README.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
-This repository contains Syncfusion PDF library examples that explains how to create PDF/A-4 conformance and convert normal PDF to PDF/A-4 conformance document.
 
-## Blog reference
-[Create PDF/A-4 Documents for Long-Term Archiving in C#](https://www.syncfusion.com/blogs/post/create-pdf-a-4-documents-for-long-term-archiving-in-c.aspx)
+The Syncfusion [.NET PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows you to create, edit, and modify PDF documents in your .NET applications.This repository contains examples of creating PDF/A-4 conformance documents and converting PDF to PDF/A-4 document.
+
+ Sample name | Description
+------|------
+[PDF/A-4e](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_4e_conformance) |  PDF/A-4e (engineering) is intended for technical documents and replaces PDF/E format. It supports 3D models, rich media, 3D annotations, and embedded files. 
+ [PDF/A-4f](https://github.com/SyncfusionExamples/PDF_A4_conformance/blob/master/Pdf_4f_conformance) | PDF/A-4f allows us to embed any files in PDF documents as attachments similar to the PDF/A-3 conformance level.
```
Comment (ghost) [inline README.md:8]: Add (.) in each sentence

Comment (sameerkhan001) [reply README.md:8]: Added

**Review by praveenHari24 — APPROVED** [review]

**Pull request merged**